pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -122,8 +126,10 @@ func NewPodController(
 		podUpdateSubscriber:   podUpdateSubscriber,
 		ovsBridgeClient:       ovsBridgeClient,
 		interfaceStore:        ifaceStore,
+		primaryInterfaceStore: primaryInterfaceStore,
```

Comment:
We no long need topass `primaryInterfaceStore` to `reconcileSecondaryInterfaces`, as it is now a member of PodController. Could you remove the argument from `reconcileSecondaryInterfaces`?

Reply:
done